tests/sentry/models/test_organization.py

```diff
@@ -58,6 +82,30 @@ def test_slugify_long_org_names(self):
         assert len(org2.slug) > len("stove-electrical-and-cateri-")
         assert org.slug != org2.slug
 
+    def test_slugify_long_org_names_within_region(self):
+        if SiloMode.get_current_mode() != SiloMode.REGION:
+            return
```

Comment:
I toyed around with the suggestion to change the test decorator instead of adding test -> region skips, but it started to look like a heavier lift than I wanted. Happy to revisit this if we think there's a better way to perform this testing.